docs/releaseNotes.yml

```diff
@@ -51,11 +59,25 @@ items:
           Users who rely on the specific ECMAScript Regex syntax will need to rewrite their regular
           expressions with RE2 syntax before upgrading to $productName$ 3.0.0.
 
-          Note that the <code>AMBASSADOR_ENVOY_API_VERSION</code> environment variable is now a
-          misnomer, as it no longer configures which xDS API version is used, but it still affects
-          what the default protocol used for a <code>TracingService</code> that points at Zipkin.
+          Note that the <code>AMBASSADOR_ENVOY_API_VERSION</code> environment variable is remove, 
+          it no longer configures which xDS API version is used.
+      - title: Support for protocol_version v2 is removed
+        type: change
+        body: >-
+          With the ugprade to Envoy 1.22, $productName$ no longer supports the V2 transport protocol.
+          The <code>AuthService</code>, <code>LogService</code> and the <code>RateLimitService</code> 
+          will only support the v3 protocol_version. If upgrading from a previous version you will want 
+          to explicitly set <code>protocol_version: "v3"</code> before upgrading to $productName$ 3.Y.
+          If protocol_version is not specified, the new default will be <code>v3</code>.
+      - title: Zipkin no longer supports HTTP_JSON_V1 for collector_endpoint_version
+        type: change
+        body: >-
+          With the upgrade to Envoy 1.22, the <code>zipkin</code> driver for the <code>TraceService</code>
+          no longer supports setting the <code>collector_endpoint_version: HTTP_JSON_V1</code>. This was removed
+          in Envoy 1.20 - <a href="https://github.com/envoyproxy/envoy/commit/db74e313b3651588e59c671af45077714ac32cef" />. 
+          The new default will be <code>collector_endpoint_version: HTTP_JSON</code>.
```

Comment:
It would probably be good to add `, regardless of the `AMBASSADOR_ENVOY_API_VERSION` environment variable` to the end of that sentence.